README_BUILD.md

```diff
@@ -1,45 +1,56 @@
-Building DSC Resources
-======================
-## Quick-start
+# Building DSC Resources
 
-~~~
-# install the bundler gem
-gem install bundler
+The `dsc` module uses rake tasks to parse the DSC Resource MOF schema files to generate Puppet types. This process pulls the DSC Resources from the github [PowerShell/DSCResources](https:/github.com/powershell/DSCResources) repo using git. This is a point-in-time snapshot, so anytime any of those DSC Resources are updated the build process needs to be run again to pull in the updated resources and parse them. If one wants to use non Microsoft DSC Resources, then you should follow the [Build Custom DSC Resource Types](#building-puppet-types-from-custom-dsc-resources) instructions.
```

Comment:
Oh, yeah, our style is to talk to users in the second person: "If you want to..." It's friendlier, and much easier to read and parse. Although in this case, I would actually just change it to more like:

"To use non-Microsoft DSC Resources, follow the ..."